### ISSUE TITLE
Feature request: Candle effect that is individual on the LED's

### ISSUE THREAD

**Issue by fribse** [Aircoookie/WLED#541]
WLED: 0.9.0-b1

[Candleholder with WS2812B LEDs](https://community.home-assistant.io/t/modifying-my-swedish-candleholder/159590?u=fribse)

The current 'candle' effect is quite nice, but it looks really odd that all the 'candles' flicker synchroniusly.
The effect would be much improved if they flickered individually.

**Comment by fishbone-git** [comment]
The effcalled Fire Flicker

**Comment by fishbone-git** [comment]
The effect you want is called Fire Flicker

**Comment by Aircoookie** [comment]
That candleholder is a really cool project! Fire flicker would definitely work, but it's effect is simpler and not as nice looking as Candle. I will work on a multi candle mode soon! For now, you could also create segments (1 for each LED) and set each segment to Candle. Save in preset slot 16 to easily recall the effect :)

**Comment by fribse** [comment]
@fishbone-git  i did look at the fireflicker, but it didn't look as nice :-)
@aircookie, thankyou for the nice words about my project, the wife is starting to accept it, especially when I told here about the candle effect. Hey, I will look into the 'segment' thing, I thought they were synchronius as well :smile:

**Comment by fribse** [comment]
@Aircoookie Ok, I got the segments set up, as it can 'only' handle 10, I set them up with a few LED's in each.
I can save them all to slot 16, but I can't make it put the effect on all of them when it's 'candle' I can do some of the other effects.
If I then apply the slot 16, it will do 'candle' on segment 0, but the others will just show solid.
Then if I click slot 1 after the slot 16 (slot 1 has a candle effect set up for 1 segment), it will start doing candle on all of the segments, and they are now out of sync.
Did I do something incorrcetly, or is it a bug?
